landbosse/model/CostModule.py

```diff
@@ -1,9 +1,30 @@
-class SharedOutputMixin:
+import math
+
+class CostModule:
     """
-    This is a mixin for shared code among modules that create output for
-    spreadsheets and other reports.
+    This is a super class for all other cost modules to import
+    that provides shared methods for outputs from results and
+    mobilization cost calculations.
     """
 
+    def mobilization_cost(self, turbine_rating):
```

Comment:
This method ended being shared in more than one module so we refactored it to here.

Reply:
Removed distributed capability from this PR.